src/cli.rs

```diff
@@ -259,6 +259,9 @@ mod tests {
     };
 
     #[tokio::test]
+    // Currently pretty flaky, we still have an agones integration test for this
+    // use-case that will catch it.
+    #[ignore]
```

Comment:
Making a note that I should look at this flake as well with the work I'm doing on #877 - is likely the same I expect.